client/src/store/profile/reducers.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    GET_USER_PENDING,
+    GET_USER_SUCCESS,
+    GET_USER_ERROR,
+    END_LOADING,
+} from './actionTypes';
+
+interface IState {
+
+    user: any;
```

Comment:
client/src/components/Profile/Profile.tsx

export interface IUserData {
        first_name: string;
        last_name: string;
        avatar: string;
    }

and use it everywhere

Reply:
Done